app/uk/gov/hmrc/hec/models/HECTaxCheck.scala

```diff
@@ -19,14 +19,16 @@ package uk.gov.hmrc.hec.models
 import play.api.libs.json.{Json, OFormat}
 
 import java.time.{LocalDate, ZonedDateTime}
+import java.util.UUID
 
 final case class HECTaxCheck(
   taxCheckData: HECTaxCheckData,
   taxCheckCode: HECTaxCheckCode,
   expiresAfter: LocalDate,
   createDate: ZonedDateTime,
   isExtracted: Boolean,
-  correctiveAction: Option[CorrectiveAction]
+  correctiveAction: Option[CorrectiveAction],
+  fileCorrelationId: Option[UUID] = None
```

Comment:
just to be sure we're not unsetting it when we shouldn't be it'd be safer to not have this default argument here I think

Reply:
okay

Reply:
i added None, so that it doesn't break the frontend and other test cases. Should i make change in FE alongwith this? else i wont be able to merge it

Reply:
ok, just realised , we can add None  val taxCheck     =
      HECTaxCheck(taxCheckData, taxCheckCode, expiryDate, createDate, false, None, None)
      here and it won't break anything from FE

Reply:
[updated](https://github.com/hmrc/hec/commit/9b13a7e761067483afa9751e0435afbcc4908c30)